src/middlewares/product.middlewares.ts

```diff
@@ -45,7 +44,7 @@ export const isProductExist = async (
 	if (!product) {
 		return sendResponse(res, 404, "NOT FOUND", "product is not found");
 	}
-	if (product?.dataValues.productStatus === "Unavailable") {
+	if (product?.dataValues.isAvailable === false) {
```

Comment:
remove false accession and use  (!) operator 
if  (!product?.dataValues.isAvailable ) {
}